scripts/check_frontend_coverage.py

```diff
@@ -484,7 +484,7 @@ def check_coverage_changes():
                     '\033[1m{}\033[0m seems to be fully covered!'
```

Comment:
Sure, that sounds good!

Reply:
@nithusha21 @DubeySandeep  instead of doing this, we want to rename file? 
check_frontend_coverage.py--> check_frontend_test_coverage.py?
check_frontend_coverage_test.py remain as it is?
am i right?
Can i push commit with file rename?

Reply:
Correct